### PR TITLE
chore: bump vault vyper version

### PR DESCRIPTION
**Pull request by pandadefi**

### CODE REVIEW

File: tests/functional/conftest.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import pytest
 
 from brownie import Token, TokenNoReturn
+from eth_account import Account
```
Comment (fubuloubu) [inline tests/functional/conftest.py:4]: What's this import for?

**Review by fubuloubu — APPROVED** [review]

File: tests/functional/conftest.py (hunk 1 above)
Comment (pandadefi) [reply tests/functional/conftest.py]: removed

**Comment by fubuloubu** [comment]
Is this Vault upgrade going to be released soon?

**Comment by pandadefi** [comment]
We should bring the discussion to the team, since there is some gas saving we can push a new version for future vaults

**Pull request merged**